### PR TITLE
[ZeroFox] Add filter to fetch only escalated alerts

### PR DESCRIPTION
**Pull request by figarrido**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- It adds a missing command to the yml file
- It adds a new config param to fetch only escalated alerts
- It fixes an issue with transforming the body of the requests to strings

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by MichaelYochpaz** [comment]
Hi @figarrido, thank you for your contribution.
Your PR review will be slightly delayed since it is the holiday season in Israel, and we have a holiday this weekend (21th-25th of September), I apologize in advance.

**Review by MichaelYochpaz — COMMENTED** [review]
Hey, looks good, just a minor note :)

File: Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.yml [hunk 1]
```diff
@@ -12,6 +12,11 @@ configuration:
   name: credentials
   required: true
   type: 9
+- display: Escalated Alerts Only
```
[the remark this answers is not in the document]
Comment (figarrido) [reply Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.yml]: Updated 👍🏼

**Review by MichaelYochpaz — DISMISSED** [review]
Looks good, good job :)

File: Packs/ZeroFox/ReleaseNotes/1_2_1.md
Comment (MichaelYochpaz) [inline Packs/ZeroFox/ReleaseNotes/1_2_1.md:1]: Missed that file. Why bump another version in the same PR?
I'm assuming this is by mistake?
You can just remove this file and update the version (to 1.2.0) on the `pack_metadata.json` file.

**Review by MichaelYochpaz — APPROVED** [review]
Demo went good, good job :)

**Pull request merged**